bazel/repository_locations.bzl

```diff
@@ -830,8 +868,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WebAssembly for Proxies (C++ host implementation)",
         project_desc = "WebAssembly for Proxies (C++ host implementation)",
         project_url = "https://github.com/proxy-wasm/proxy-wasm-cpp-host",
-        version = "b7d3d13d75bb6b50f192252258bb9583bf723fa4",
-        sha256 = "ae639f94a80adbe915849bccb32346720c59a579db09918e44aefafed6cbb100",
+        version = "4741d2f1cd5eb250f66d0518238c333353259d56",
+        sha256 = "30fc4becfcc5a95ac875fc5a0658a91aa7ddedd763b52d7810c13ed35d9d81aa",
```

Comment:
Suggest updating to the latest commit
```
        version = "eceb02d5b7772ec1cd78a4d35356e57d2e6d59bb",
        sha256 = "ae9d9b87d21d95647ebda197d130b37bddc5c6ee3e6630909a231fd55fcc9069",
        release_date = "2020-11-10",
```

Reply:
The latest commit requires a ton of other changes (see: https://github.com/envoyproxy/envoy/pull/13753).